### DISCUSSION TITLE
How to run my program on docker

### DISCUSSION THREAD

**Discussion by kymind** [NVIDIA/NVFlare#1856] [Q&A]
I want to run with docker, but the process would be killed when i execute it.

**Comment by Kane-ouvic** [comment]
Maybe this reference is helpful
https://nvflare.readthedocs.io/en/main/getting_started.html

notice: maybe u have to add while(1) in command, in order to prevent the container terminated when it execute the last code.

**Reply by chesterxgchen** [reply]
if you try with 2.4.0rc1, the new POC command has run docker in detached the mode. You will need to pre-build and docker image with nvflare pre-installed.  then running nvflare on local machine via poc.

**Reply by YuanTingHsieh** [reply]
@kymind does this help^
